tests/test_dataset_registry.py

```diff
@@ -6,8 +6,8 @@
 
 from composer.datasets import (ADE20kDatasetHparams, BratsDatasetHparams, C4DatasetHparams, CIFAR10DatasetHparams,
                                COCODatasetHparams, DataLoaderHparams, DatasetHparams, GLUEHparams,
-                               ImagenetDatasetHparams, LMDatasetHparams, MNISTDatasetHparams, SyntheticHparamsMixin,
-                               WebDatasetHparams)
+                               ImagenetDatasetHparams, LMDatasetHparams, MNISTDatasetHparams, StreamingDatasetHparams,
```

Comment:
Where is `StreamingDatasetHparams` defined?

Should it be a mixin (similar to SyntheticHparamsMixin) rather than a dataset type?

Reply:
I think I was planning on removing `StreamingDatasetHparams`, I may have just forgotten to update this test.